docs/usage/logging.md

```diff
@@ -65,6 +67,17 @@ Examples:
     ```{job="systemd-combine-journal",nodename="ip-10-222-31-182.eu-central-1.compute.internal"} | unpack | unit="cloud-config-downloader.service" |~ "last execution was"```
 > **_NOTE:_** `{job="systemd-combine-journal",nodename="<node name>"}` stream [pack](https://grafana.com/docs/loki/latest/clients/promtail/stages/pack/) all logs from systemd services except `docker`, `containerd`, `kubelet` and `kernel`. To filter those log by unit you have to [unpack](https://grafana.com/docs/loki/latest/logql/log_queries/#unpack) them first.
 
+4. You want to get events from the shoot `kube-system` namespace and from the shoot control plane.
+  - Logs from the shoot `kube-system` namespace, generated from the `kubelet` and related to the `node-problem-detector`:
+
+  ```{job="event-logging"} | unpack | origin_extracted="Shoot",source="kubelet",object=~".*node-problem-detector.*"```
+
+  - Logs from the MCM in the shoot `control-plane` in the seed:
+
+  ```{job="event-logging"} | unpack | origin_extracted="Seed",source=~".*machine-controller-manager.*"```
```

Comment:
As showed you in a call, this query didn't work for me - the `origin_extracted` label was not there.

Reply:
It was fixed.

Reply:
Can you explain what was the issue for containerd Nodes and what we did to fix it? Thanks in advance!

Reply:
The containerd parser was using escaped decoding instead of json.
Thus the log section is not escaped as nested json but simply it was escaped with one `\`.